### PR TITLE
amp-sidebar - remove experimental v 1.0, all features already merged with 0.1

### LINKED ISSUE #9803: Clean up amp-sidebar v1 experiment after launch

### PR DESCRIPTION
**Pull request by aghassemi**
Closes #9803

### CODE REVIEW

File: extensions/amp-sidebar/1.0/amp-sidebar-1.0.md [hunk 1]
```diff
@@ -1,255 +0,0 @@
-<!---
```
Comment (cathyxz) [inline extensions/amp-sidebar/1.0/amp-sidebar-1.0.md]: Does this PR also delete the entire sidebar component...?

File: extensions/amp-sidebar/1.0/amp-sidebar.js [hunk 2]
```diff
@@ -1,365 +0,0 @@
-/**
```
Comment (cathyxz) [inline extensions/amp-sidebar/1.0/amp-sidebar.js]: Is this file actually supposed to be deleted?

File: extensions/amp-sidebar/1.0/amp-sidebar-1.0.md (hunk 1 above)
Comment (cathyxz) [reply extensions/amp-sidebar/1.0/amp-sidebar-1.0.md]: nvm, it's 1.0.

**Comment by torch2424** [comment]
Thanks @aghassemi ! 👍

**Pull request merged**